<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>829</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_setting">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>710</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>설정</string>
    </property>
    <property name="whatsThis">
     <string>설정</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/setting.png</normaloff>images/setting.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_improve">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>710</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>인식이 잘 되지 않는 문자는 직접 학습시킬 수 있습니다.</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string>인식 개선</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_help">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>710</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>앱에서 사용하는 지화의 형태를 확인할 수 있습니다.</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string>도움말</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam_you">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(200, 200, 200)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cam_me">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(150, 150, 150)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_me">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>510</y>
      <width>550</width>
      <height>45</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:30px</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>내용</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_you">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>570</y>
      <width>550</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:30px</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>내용</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_ip">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>660</y>
      <width>530</width>
      <height>40</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:25px</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="placeholderText">
     <string>ip 입력</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_call">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>630</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>연결</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/connect.png</normaloff>
      <normalon>images/disconnect.png</normalon>images/connect.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>640</y>
      <width>441</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>통화할 IP를 입력해주세요. 미입력시 연습모드로 실행됩니다</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>515</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:20pt</string>
    </property>
    <property name="text">
     <string>나  : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:15pt</string>
    </property>
    <property name="text">
     <string>상대 : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_engNum">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>430</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>앱에서 사용하는 지화의 형태를 확인할 수 있습니다.</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(250, 250, 250, 200);
font:10pt &quot;나눔바른고딕&quot;;</string>
    </property>
    <property name="text">
     <string>영어/숫자</string>
    </property>
    <property name="iconSize">
     <size>
      <width>21</width>
      <height>21</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>pushButton_setting</tabstop>
  <tabstop>pushButton_improve</tabstop>
  <tabstop>pushButton_help</tabstop>
  <tabstop>textEdit_me</tabstop>
  <tabstop>textEdit_you</tabstop>
  <tabstop>textEdit_ip</tabstop>
  <tabstop>pushButton_call</tabstop>
  <tabstop>pushButton_engNum</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>textEdit_ip</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>pushButton_call</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>707</y>
    </hint>
    <hint type="destinationlabel">
     <x>612</x>
     <y>695</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
